scripts.js

```diff
@@ -13,50 +14,43 @@ addCopy.classList.add("fas", "fa-copy", "copy", "icon", "push");
 addDelete.classList.add("fas", "fa-times", "delete", "icon", "edit");
 
 // focus curson in input on load
-window.onload = getInput.focus();
+window.onload = inputElement.focus();
 
 // add return key listen and call addItem function
-getInput.addEventListener('keydown', function() {
+inputElement.addEventListener('keydown', function() {
   if (event.code == 'Enter') {
```

Comment:
Please use the [strict equality comparator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#Strict_equality_using)